hoymiles_mqtt/__main__.py

```diff
@@ -160,13 +178,29 @@ def _parse_args() -> argparse.Namespace:
         env_var='COMM_RECONNECT_DELAY',
         help="Additional low level modbus communication parameter - delay in milliseconds before reconnecting.",
     )
+    cfg_parser.add(
+        '--log-level',
+        required=False,
+        type=str,
+        default=logging.WARNING,
```

Comment:
Should it be default='WARNING' ?

Reply:
It seems that the test is failing due to this.

Reply:
Changed as requested.